### DISCUSSION TITLE
Feature: disable caching globally through config

### DISCUSSION THREAD

**Discussion by BrandonOldenhof** [ryangjchandler/blade-cache-directive#9] [Ideas]
Hello!

Small feature request here: it would be great if we could enable/disable caching through the config file, or through the .env directly.
Caching is incredibly annoying while developing locally but we can't disable it yet.
Thoughts?

@ryangjchandler PS: trying to create a new Issue in the repo redirects to a 404-page. Not too strange because the URL it tries to go to is https://github.com/vendor_slug/package_slug/discussions/new?category=ideas.

**Comment by ryangjchandler** [comment]
@BrandonOldenhof Yeah, this is a reasonable request actually. Let me take a look in a minute!

**Comment by ryangjchandler** [comment]
@BrandonOldenhof I've released v0.4.0 with support for this, please check the README for instructions. Thanks!
